src/services/dashboards/resource.js

```diff
@@ -330,6 +331,438 @@ const switchToTopicCentric = (input) => {
+      FROM ${tblNames.createdTopicsTempTableName}  t
+        JOIN ${tblNames.createdAroTopicsTempTableName} arot
+        ON t.id = arot."topicId"
+        JOIN ${tblNames.createdFlatResourceTempTableName} f
+        ON arot."activityReportId" = f.id
+        GROUP BY f.id, t.name, f."rollUpDate"
+        ORDER BY t.name, f."rollUpDate" ASC
+    ),
+    topicsperdate AS
+    (
+    SELECT
+      "name",
+      "rollUpDate",
+      SUM("resourceCount") AS "resourceCount"
+    FROM topics
+    GROUP BY "name", "rollUpDate"
+    ),
+    totals AS
+    (
+      SELECT
+        name,
+        SUM("resourceCount") AS "totalCount"
+      FROM topicsperdate
+      GROUP BY name
+      ORDER BY SUM("resourceCount") DESC
+    ),
+    series AS
+    (
+      SELECT * FROM ${tblNames.createdFlatResourceHeadersTempTableName}
+    )
+    SELECT
+      d.name,
+      to_char(s."date", 'Mon-YY') AS "rollUpDate",
+      s."date",
+      coalesce(t."resourceCount", 0) AS "resourceCount",
+      tt."totalCount"
+    FROM ${tblNames.createdTopicsTempTableName} d
+    JOIN series s
+      ON 1=1
+    JOIN totals tt
+      ON d.name = tt.name
+    LEFT JOIN topicsperdate t
+      ON d.name = t.name AND to_char(s."date", 'Mon-YY') = t."rollUpDate"
+    ORDER BY 1, 3 ASC;`;
+  return sequelize.query(
+    topicUseSql,
+    {
+      type: QueryTypes.SELECT,
+      transaction,
+    },
+  );
+}
+
+function getOverview(tblNames, totalReportCount, transaction) {
+  // - Number of Participants -
+  const numberOfParticipants = sequelize.query(`
+  WITH ar_participants AS (
+    SELECT
+    id,
+    "numberOfParticipants"
+    FROM ${tblNames.createdFlatResourceTempTableName} f
+    GROUP BY id, "numberOfParticipants"
+  )
+  SELECT
+         SUM("numberOfParticipants") AS participants
+  FROM ar_participants;
+  `, {
+    type: QueryTypes.SELECT,
+    transaction,
+  });
+
+  const numberOfRecipSql = `
+  WITH ars AS (
+    SELECT
+    DISTINCT id
+    FROM ${tblNames.createdFlatResourceTempTableName} f
+  ), recipients AS (
+    SELECT
+      DISTINCT r.id
+    FROM ars ar
+    JOIN "ActivityRecipients" arr
+      ON ar.id = arr."activityReportId"
+    JOIN "Grants" g
+      ON arr."grantId" = g.id AND g."status" = 'Active'
+     JOIN "Recipients" r
+      ON g."recipientId" = r.id
+  )
+  SELECT
+         count(r.id) AS recipients
+  FROM recipients r;
+  `;
+
+  // - Number of Recipients -
+  const numberOfRecipients = sequelize.query(numberOfRecipSql, {
+    type: QueryTypes.SELECT,
+    transaction,
+  });
+
+  // - Reports with Resources Pct -
+  const pctOfResourcesSql = `
+  SELECT
+    count(DISTINCT "activityReportId")::decimal AS "reportsWithResourcesCount",
+    ${totalReportCount}::decimal AS "totalReportsCount",
+    CASE WHEN ${totalReportCount} = 0 THEN
+      0
+    ELSE
+      (round(count(DISTINCT "activityReportId")::decimal / ${totalReportCount}::decimal, 4) * 100)::decimal
```

Comment:
only one of the values needs to be cast to a decimal to get a decimal value after the division, there should be no need to cast the result to a decimal also
```suggestion
      (count(DISTINCT "activityReportId") / ${totalReportCount} * 100)::decimal(4,2)
```